### PR TITLE
Some Patches

### PR DESCRIPTION
**Pull request by IRacle1**
I rewrite `AntiCheatSpeedFix` and `MuteHandlerClear` patches, everything worked on my tests
(removed null check in `MuteHandlerClear`, i don't think it's necessary)

### CODE REVIEW

**Comment by IRacle1** [comment]
@louis1706 
```cs
var player = Player.Get(__instance._hub);
speed *= __instance.staminaController.AllowMaxSpeed ? player.RunningSpeed : player.WalkSpeed;
```
Did I understand correctly that your patch adds only lines above?(and remove base game setter)

**Comment by louis1706** [comment]
> @louis1706 
> ```cs
> var player = Player.Get(__instance._hub);
> speed *= __instance.staminaController.AllowMaxSpeed ? player.RunningSpeed : player.WalkSpeed;
> ```
> Did I understand correctly that your patch adds only lines above?(and remove base game setter)

Yes i thinks i have does that

**Comment by IRacle1** [comment]
Okay

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -338,12 +338,12 @@ public float RunningSpeed
         /// <summary>
         /// Gets or sets walkSpeed of the player.
         /// </summary>
-        public float WalkSpeed
+        public float WalkingSpeed
```
Comment (BuildBoy12) [inline Exiled.API/Features/Player.cs:341]: Changing this is a breaking change.

File: Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs [hunk 2]
```diff
@@ -7,127 +7,69 @@
...
+    using NorthwoodLib.Pools;
 
-    using PlayableScps;
-
-    using UnityEngine;
+    using static HarmonyLib.AccessTools;
 
     /// <summary>
-    /// Fixes AntiCheat making rollback the player when he get there speed changed"/>.
+    /// Fixes position desync when player speed changed"/>.
```
Comment (BuildBoy12) [inline Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs:22]: Remove unnecessary "/>

File: Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs [hunk 3]
```diff
@@ -7,127 +7,69 @@
...
+    /// Adds <see cref="Player.RunningSpeed"/> and <see cref="Player.WalkingSpeed"/> implementation.
     /// </summary>
     [HarmonyPatch(typeof(FirstPersonController), nameof(FirstPersonController.GetSpeed))]
     public static class AntiCheatSpeedFix
```
Comment (BuildBoy12) [inline Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs:26]: Should probably be made internal to meet the current transpiler style

**Review by BuildBoy12 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (IRacle1) [reply Exiled.API/Features/Player.cs:341]: Yamato made a PR on these properties about a week ago

Comment (IRacle1) [reply Exiled.API/Features/Player.cs:341]: So I think that can be changed.

File: Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs (hunk 2 above)
Comment (IRacle1) [reply Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs]: Done

File: Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs (hunk 3 above)
Comment (IRacle1) [reply Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs]: Done

**Review by BuildBoy12 — APPROVED** [review]

**Pull request merged**